### ISSUE TITLE
Config file for fine-tune zjumocap dataset

### ISSUE THREAD

**Issue by weiyichang** [zju3dv/ENeRF#12]
Hi,

Thanks for sharing this nice work.

I would like know that how to fine-tune model for zjumocap dataset.

I have modified the config from zjumocap_eval.yaml, but the results are worse than the pre-trained model.

Do you have suggestions ? 

Thanks !!!

**Comment by haotongl** [comment]
I'll check it out, please be patient.

**Comment by haotongl** [comment]
I have updated the corresponding config file and README. If everything is set up, the following results can be obtained by `tensorboard --logdir=$workspace/record --bind_all`: 
![image](https://user-images.githubusercontent.com/45787711/211160571-1b05c6f9-d769-4a7f-84e9-81fe3023daa5.png)

**Comment by weiyichang** [comment]
Thanks a lot, I will try it.
One more thing I would like to confirm is about the epoch number in train part, 
Is it necessary to train with 100 epoches ? 
(I will try it by myself, but I also would like to know your experience)
 Thanks !!!

**Comment by haotongl** [comment]
From my experience, there will be no obvious visual improvement after several (5) epochs. But the metric may fully converge after 40 epochs.

**Comment by weiyichang** [comment]
Thanks for your quick response.
I will try it :)
